tests/test_template_loader.py

```diff
@@ -0,0 +1,76 @@
+from pathlib import Path
```

Comment:
I've split the `test_autodiscover.py` file and move the tests for template loader into `test_template_loader.py`.